js/components/ui/Modal.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import * as React from 'react'
+import RNModal, { ModalProps } from 'react-native-modal'
+import mapPropsToStyleNames from 'native-base/src/Utils/mapPropsToStyleNames.js'
+import { connectStyle } from 'native-base-shoutem-theme'
+
+type Props = ModalProps
+
+/**
+ * A customized react-native-modal component.
+ * The class is connected with the native-base StyleProvider using the `connectStyle(...)` method.
+ */
+class Modal extends React.Component<Props> {
+  render(): React.Node {
```

Comment:
it's not clear why we need to qualify the render method with `: React.Node`, isn't this implied in the flow type of the React.Component class ? (disclaimer: I don't know flow :)

Reply:
The problem here is the rule in eslint config 'flowtype/require-return-type', with this rule active you must also annotate the return type of the render() method.